pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -215,6 +214,13 @@ stdenv.mkDerivation {
 
         # journalctl --grep requires libpcre so lets provide it
         { name = "libpcre2-8.so.0"; pkg = pcre2; }
+
+        # unclear
+        { name = "libfido2.so.1"; pkg = null; }
+        { name = "libfido2.so.1"; pkg = null; }
+        { name = "libtss2-esys.so.0"; pkg = null; }
+        { name = "libtss2-rc.so.0"; pkg = null; }
+        { name = "libtss2-mu.so.0"; pkg = null; }
```

Comment:
@arianvp I assume you experimented with all the FIDO and TPM stuff. I assume that's useful to have, at least in the non-minimal version of systemd?

Reply:
I've been planning to. I think we should add it as it's one of the cooler features of 248.